.github/workflows/ci.yml

```diff
@@ -375,36 +375,21 @@ jobs:
           touch gib-impacted.log
           cat <<EOF > .github/test-matrix.yaml
           include:
-            - { modules: core/trino-main }
-            - { modules: testing/trino-tests }
-            - { modules: plugin/trino-raptor-legacy }
-            - { modules: plugin/trino-accumulo }
-            - { modules: plugin/trino-cassandra }
-            - { modules: plugin/trino-clickhouse }
-            - { modules: plugin/trino-delta-lake }
-            - { modules: plugin/trino-hive }
-            - { modules: plugin/trino-hive, profile: test-parquet }
-            - { modules: plugin/trino-hive, profile: test-failure-recovery }
-            - { modules: plugin/trino-hive, profile: test-fault-tolerant-execution }
-            - { modules: plugin/trino-elasticsearch }
-            - { modules: plugin/trino-elasticsearch, profile: test-stats }
-            - { modules: plugin/trino-mongodb }
-            - { modules: plugin/trino-kafka }
-            - { modules: plugin/trino-pinot }
-            - { modules: plugin/trino-redis }
-            - { modules: plugin/trino-mysql }
-            - { modules: plugin/trino-postgresql }
-            - { modules: plugin/trino-sqlserver }
-            - { modules: plugin/trino-memsql }
-            - { modules: plugin/trino-oracle }
-            # TODO: re-enable kudu tests after this issue is resolved: https://github.com/trinodb/trino/issues/11203
-            # - { modules: plugin/trino-kudu }
-            - { modules: plugin/trino-druid }
-            - { modules: plugin/trino-iceberg }
-            - { modules: plugin/trino-iceberg, profile: test-failure-recovery }
-            - { modules: [ plugin/trino-phoenix, plugin/trino-phoenix5 ] }
-            - { modules: [ client/trino-jdbc, plugin/trino-base-jdbc, plugin/trino-thrift, plugin/trino-memory ] }
-            - { modules: plugin/trino-bigquery }
+            - { modules: plugin/trino-kudu }
```

Comment:
Will revert before merging.

Reply:
reminder. note you do like `- { modules: plugin/trino-kudu, i: 1 }` and so on to get GHA not rate-limit you because it appears same job is being submitted multiple times.

Reply:
Had to modify `build-matrix-from-impacted.py` to support this because the git incremental builder CI changes

Reply:
cc: @nineinchnick

Reply:
https://github.com/trinodb/trino/pull/11446